scripts/west_commands/runners/jlink.py

```diff
@@ -138,8 +108,34 @@ def dev_id_help(cls) -> str:
     def tool_opt_help(cls) -> str:
         return "Additional options for JLink Commander, e.g. '-autoconnect 1'"
 
+    @staticmethod
+    def default_jlink():
+        global DEFAULT_JLINK_EXE
+
+        if sys.platform == 'win32':
+            # JLink.exe can collide with the JDK executable of the same name
+            # Locate the executable using the registry
+            try:
+                import winreg
+                # Note that when multiple JLink versions are installed on the
+                # machine this points to the one that was installed
+                # last, and not to the latest version.
+                key = winreg.OpenKeyEx(winreg.HKEY_CURRENT_USER,
+                                       r"Software\SEGGER\J-Link")
+                DEFAULT_JLINK_EXE = Path(winreg.QueryValueEx(key,
+                                            "InstallPath")[0]) / "JLink.exe"
```

Comment:
Weird indentation here

Reply:
How so weird? I am placing the params  to QueryValueEx on the line below, what would be better?

Reply:
@pdgendt updated: I used something similar to what `ruff` proposed.